legend-engine-language-pure-compiler/src/main/java/org/finos/legend/engine/language/pure/compiler/toPureGraph/ValueSpecificationBuilder.java

```diff
@@ -132,7 +134,18 @@ public ValueSpecification visit(org.finos.legend.engine.protocol.pure.v1.model.v
     @Override
     public ValueSpecification visit(org.finos.legend.engine.protocol.pure.v1.model.valueSpecification.raw.PackageableElementPtr packageableElementPtr)
     {
-        org.finos.legend.pure.m3.coreinstance.meta.pure.metamodel.PackageableElement packageableElement = this.context.resolvePackageableElement(packageableElementPtr.fullPath, packageableElementPtr.sourceInformation);
+        PackageableElement packageableElement = this.context.resolvePackageableElement(packageableElementPtr.fullPath, packageableElementPtr.sourceInformation);
+        if (packageableElement instanceof Root_meta_pure_runtime_PackageableRuntime)
```

Comment:
Could there be a better place to convert PackageableRuntime to an actual Runtime?